projects/plugins/jetpack-mu-wpcom-plugin/composer.json

```diff
@@ -0,0 +1,47 @@
+{
+	"name": "automattic/jetpack-mu-wpcom-plugin",
+	"description": "Test plugin for the jetpack-mu-wpcom package",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-mu-wpcom": "@dev"
+	},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"php -dpcov.directory=. ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"autorelease": true,
+		"autotagger": true,
+		"beta-plugin-slug": "jetpack-mu-wpcom-plugin",
+		"mirror-repo": "Automattic/jetpack-mu-wpcom-plugin",
+		"release-branch-prefix": "jetpack"
```

Comment:
@anomiex Think I've confused myself here, should `release-branch-prefix` be `jetpack` or `jetpack-mu-wpcom-plugin`?

Reply:
Since I'm guessing we'll still want to release the plugin alongside Jetpack to update the package on wpcom and to release it for wpcomsh→Atomic. So "jetpack".